overrides/default.nix

```diff
@@ -4058,6 +4078,7 @@ lib.composeManyExtensions [
         let
           # Watchfiles does not include Cargo.lock in tarball released on PyPi for versions up to 0.17.0
           getRepoHash = version: {
+            "1.0.4" = "sha256-VoYV13GsTaAWoLcSfuadLR2l8Xbn0MEd/Uh9EP/DgjE=";
```

Comment:
If it isn't directly related with ruff then it should go in a separate commit as well.